### PR TITLE
HMOS SlideActionView Library

### PR DESCRIPTION
**Pull request by priyankabhadrappanavar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,8 +64,8 @@ actionView.setListener(new SlideActionView.SlideActionListener() {
     // slid right
   }
 });
-```
+ ```
```
Comment (samuel-abhishek) [inline README.md:68]: Clearly mention the part in future-work as discussed.

File: slideactionview/src/main/java/me/jfenn/slideactionview/SlideActionView.java [hunk 2]
```diff
@@ -1,98 +1,107 @@
...
+        selectionRadius = DimenUtils.dpToPx(getContext(), 42);
+        rippleRadius = DimenUtils.dpToPx(getContext(), 140);
         selected = new AnimatedFloat(0);
         ripples = new HashMap<>();
-
         normalPaint = new Paint();
-        normalPaint.setStyle(Paint.Style.FILL);
-        normalPaint.setColor(Color.GRAY);
+        normalPaint.setStyle(Paint.Style.FILL_STYLE);
+        Color hmosColor = SlideActionView.changeParamToColor(Color.RED.getValue());
```
Comment (samuel-abhishek) [inline slideactionview/src/main/java/me/jfenn/slideactionview/SlideActionView.java]: Please keep the color codes similar to the library's base code..as it is hardcoded within the library. Applies to all the similar instances which are present below in the same file

File: entry/src/main/resources/base/element/string.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "string": [
+    {
+      "name": "entry_MainAbility",
```
Comment (samuel-abhishek) [inline entry/src/main/resources/base/element/string.json:4]: Remove the unnecessary strings and replace the required strings with meaningful string values.

File: README.md [hunk 4]
```diff
@@ -1,60 +1,59 @@
...
 
-For testing and experimentation purposes, a sample apk can be downloaded [here](https://jfenn.me/projects/slideactionview).
+## Source
+Inspired by [fennifith/SlideActionView](https://github.com/fennifith/SlideActionView) - version 0.0.2
 
-|Sample App|Use in Alarmio|
-|------|------|
-|![img](./.github/images/sample.png?raw=true)|![img](./.github/images/alert.gif?raw=true)|
+## Feature
+SlideActionView is a simple widget that provides a nice slide-to-left/right interaction.
```
Comment (samuel-abhishek) [inline README.md:9]: In the feature section, mention about the OhosUtils library that you are using internally : https://github.com/applibgroup/OHOSUtils/

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please check the comments

File: slideactionview/src/main/java/me/jfenn/slideactionview/SlideActionView.java (hunk 2 above)
Comment (priyankabhadrappanavar) [reply slideactionview/src/main/java/me/jfenn/slideactionview/SlideActionView.java]: Done

**Review by samuel-abhishek — APPROVED** [review]
Okay

File: README.md (hunk 4 above)
Comment (priyankabhadrappanavar) [reply README.md:9]: Done

File: entry/src/main/resources/base/element/string.json (hunk 3 above)
Comment (priyankabhadrappanavar) [reply entry/src/main/resources/base/element/string.json:4]: Done

File: README.md (hunk 1 above)
Comment (priyankabhadrappanavar) [reply README.md:68]: Done

**Review by sharathks84 — APPROVED** [review]
Okay, approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**